src/App.scss

```diff
@@ -0,0 +1,2 @@
+@import "~bootstrap/scss/bootstrap";
+@import "~@edx/edx-bootstrap/sass/edx/theme";
```

Comment:
I think the theme should be imported before the Boostrap scss. The theme defines a bunch of variables (colors, etc.) and the Bootstrap scss uses those variables.